src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1,22 +1,21 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
-// THIS IS AUTOGENERATED FILE
-// USE ENCODINGDATA TOOL TO GENERATE SUCH FILE
+// THIS IS AN AUTOGENERATED FILE
+// IT IS GENERATED FROM A BUILDTOOLS TASK
```

Comment:
....you mean like add it to the output of the build tools task, so it gets added automatically?  The PR for which you just merged?